chainer_setup_build.py

```diff
@@ -388,6 +421,10 @@ def cythonize(extensions, force=False, annotate=False, compiler_directives={}):
 
     cython_cmdbase.extend(['--fast-fail', '--verbose', '--cplus'])
     for ext in extensions:
+        if all(not _is_changed(f) for f in ext.sources) and only_changed:
```

Comment:
What's the difference between 
`if all(not _is_changed(f) for f in ext.sources) and only_changed:`
and
`if all([not _is_changed(f) for f in ext.sources]) and only_changed:`
?